### ISSUE TITLE
Compliance/Accessability/Summaries/Add base Action column

### ISSUE THREAD

**Issue by vesselak** [bcgov/cas-compliance#150]
#### Describe the task

As created in #115 each compliance summary should have an action column with different actions based on multiple conditions.

The following accessibility findings should be addressed as noted by QA:

- [ ]  When using keyboard navigation (test by tabbing) 
- [ ] After Compliance Summaries tab, the cursor navigates to Reporting Year search field instead of Payment Summaries 
- [ ] After tabbing from Reporting Year search field the cursor is now gone and tabbing not available anymore

#### Definition of Done

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Additional context

- Add any other context about the task here.
- Or here

**Comment by vesselak** [comment]
@fviduya can you re-test this one next week and confirm if we can delete. thanks cc @patrickisaac

**Comment by patrickisaac** [comment]
@vesselak - Followed @fviduya's previous testing document, tested the A/C, and confirmed:

- [x] When using keyboard navigation (test by tabbing)
- [x] After Compliance Summaries tab, the cursor navigates to Reporting Year search field instead of Payment Summaries
- [ ] After tabbing from Reporting Year search field the cursor is now gone and tabbing not available anymore 

Unsure what @fviduya meant by the last A/C since I don't believe we have the search field functionality anymore. Other than that, everything else looks good though!

**Comment by vesselak** [comment]
Closing ticket with the comment above. The sorting will be tested as part of #260
